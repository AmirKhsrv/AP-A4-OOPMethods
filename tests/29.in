credit 42
best_book
best_book
show_sorted_shelf 83 read Romance
show_sorted_shelf 33 read Autobiography
recommend_first_approach 65
best_book
best_reviewer
credit 123
recommend_first_approach 58
