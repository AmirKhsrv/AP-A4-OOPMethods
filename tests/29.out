0.004932
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 35
Title: No Strings Attached
Genre: Romance
Author: Nicolette Day
***
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
***
id: 271
Title: Werewolf Nights
Genre: Romance
Author: Mari Hamill
***
id: 329
Title: Alias Vol. 1
Genre: Sequential Art
Author: Brian Michael Bendis
***
id: 182
Title: Arcadia Awakens
Genre: Fantasy
Author: Kai Meyer
***
id: 254
Title: Caps for Sale: A Tale of a Peddler Some Monkeys and Their Monkey Business
Genre: Childrens
Author: Esphyr Slobodkina
***
id: 287
Title: Curses Boiled Again!
Genre: Mystery
Author: Shari Randall
***
id: 119
Title: Digging Deep: Unearthing Your Creative Roots Through Gardening
Genre: Gardening
Author: Fran Sorin
***
id: 148
Title: Enchanted Inc.
Genre: Fantasy
Author: Shanna Swendson
***
id: 222
Title: Everything Is Illuminated
Genre: Fiction
Author: Jonathan Safran Foer
***
id: 250
Title: It's the Stars Will Be Our Lamps
Genre: Cultural
Author: Alan Walsh
***
id: 78
Title: Seven Sons
Genre: Dark
Author: Lili St. Germain
***
id: 54
Title: The 19th Wife
Genre: Historical
Author: David Ebershoff
***
id: 234
Title: The Great Gilly Hopkins
Genre: Fiction
Author: Katherine Paterson
***
id: 34
Title: The Maid of Fairbourne Hall
Genre: Historical
Author: Julie Klassen
***
id: 30
Title: The Tibetan Book of Living and Dying
Genre: Philosophy
Author: Sogyal Rinpoche
***
id: 199
Title: Valour and Vanity
Genre: Fantasy
Author: Mary Robinette Kowal
***
id: 382
Title: When Breath Becomes Air
Genre: Nonfiction
Author: Paul Kalanithi
***
id: 362
Title: A Season To Remember
Genre: Holiday
Author: Elizabeth Ellen Carter
***
id: 13
Title: Allegiant
Genre: Young Adult
Author: Veronica Roth
***
id: 318
Title: Blowing on Dandelions
Genre: Christian Fiction
Author: Miralee Ferrell
***
id: 400
Title: Dragon Bound
Genre: Fantasy
Author: Thea Harrison
***
id: 124
Title: Eating People is Wrong
Genre: Fiction
Author: Malcolm Bradbury
***
id: 135
Title: Fried Green Tomatoes at the Whistle Stop Cafe
Genre: Fiction
Author: Fannie Flagg
***
id: 388
Title: If I Have a Wicked Stepmother Where's My Prince?
Genre: Young Adult
Author: Melissa Kantor
***
id: 176
Title: Keturah and Lord Death
Genre: Fantasy
Author: Martine Leavitt
***
id: 35
Title: No Strings Attached
Genre: Romance
Author: Nicolette Day
***
id: 155
Title: Olive Kitteridge
Genre: Fiction
Author: Elizabeth Strout
***
id: 67
Title: Secret Daughter
Genre: Fiction
Author: Shilpi Somaya Gowda
***
id: 270
Title: Siddhartha
Genre: Classics
Author: Hermann Hesse
***
id: 342
Title: Something from the Nightside
Genre: Fantasy
Author: Simon R. Green
***
id: 8
Title: Starlet's Web
Genre: Young Adult
Author: Carla J. Hanna
***
id: 37
Title: Teaching My Mother How to Give Birth
Genre: Poetry
Author: Warsan Shire
***
id: 55
Title: Twelve Sharp
Genre: Mystery
Author: Janet Evanovich
***
id: 209
Title: Until Tuesday: A Wounded Warrior and the Golden Retriever Who Saved Him
Genre: Nonfiction
Author: Luis Carlos Montalv√°n
***
id: 63
Title: When We Wake
Genre: Young Adult
Author: Karen Healey
***
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
0.009409
id: 236
Title: Slow Burn
Genre: Romance
Author: Sam B. Morgan
Reviews:
id: 22 Rating: 0 Likes: 38 Date: February 27 2016
id: 709 Rating: 4 Likes: 2 Date: September 27 2013
id: 816 Rating: 2 Likes: 0 Date: November 29 2019
id: 1315 Rating: 5 Likes: 0 Date: October 16 2015
id: 1626 Rating: 2 Likes: 0 Date: March 28 2019
id: 1698 Rating: 3 Likes: 0 Date: July 2 2016
id: 2240 Rating: 2 Likes: 0 Date: March 4 2013
id: 2338 Rating: 3 Likes: 0 Date: March 29 2019
id: 2651 Rating: 2 Likes: 0 Date: November 25 2018
id: 2702 Rating: 5 Likes: 0 Date: October 5 2018
id: 2861 Rating: 3 Likes: 0 Date: April 11 2014
id: 4305 Rating: 5 Likes: 20 Date: May 8 2015
id: 4625 Rating: 0 Likes: 1 Date: March 8 2011
id: 5276 Rating: 5 Likes: 0 Date: September 4 2019
id: 6661 Rating: 4 Likes: 2 Date: January 11 2019
id: 6698 Rating: 3 Likes: 2 Date: December 29 2010
id: 6754 Rating: 5 Likes: 5 Date: July 23 2014
id: 7207 Rating: 4 Likes: 13 Date: June 27 2018
id: 7241 Rating: 3 Likes: 12 Date: October 20 2011
id: 7335 Rating: 1 Likes: 1 Date: April 15 2012
id: 7395 Rating: 2 Likes: 0 Date: September 9 2019
id: 7785 Rating: 2 Likes: 0 Date: January 27 2019
id: 7998 Rating: 4 Likes: 0 Date: March 19 2016
id: 8669 Rating: 3 Likes: 0 Date: May 2 2014
id: 9719 Rating: 0 Likes: 0 Date: November 24 2019
id: 9874 Rating: 2 Likes: 0 Date: August 25 2014
id: 10004 Rating: 4 Likes: 0 Date: March 25 2019
id: 10171 Rating: 4 Likes: 2 Date: February 14 2019
id: 10644 Rating: 2 Likes: 0 Date: June 25 2019
id: 10714 Rating: 4 Likes: 20 Date: June 6 2019
id: 10841 Rating: 3 Likes: 0 Date: April 27 2017
id: 10909 Rating: 5 Likes: 0 Date: April 9 2013
id: 11757 Rating: 0 Likes: 0 Date: March 27 2018
id: 12179 Rating: 5 Likes: 0 Date: September 5 2015
id: 13019 Rating: 5 Likes: 1 Date: May 21 2012
id: 13321 Rating: 4 Likes: 0 Date: May 18 2016
id: 14914 Rating: 1 Likes: 0 Date: November 17 2018
id: 16115 Rating: 5 Likes: 0 Date: April 7 2016
